src/utils/pqUtils.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import JSZip from "jszip";
+import iconv from "iconv-lite";
+import { pqCustomXmlPath } from "../constants";
+import { generateMashupXMLTemplate } from "../generators";
+
+const getBase64 = async (zip: JSZip): Promise<string> => {
+    const xmlValue = await zip.file(pqCustomXmlPath)?.async("uint8array");
+    if (xmlValue === undefined) {
+        throw new Error("PQ document wasn't found in zip");
+    }
+    const xmlString = iconv.decode(xmlValue.buffer as Buffer, "UTF-16");
+    const parser: DOMParser = new DOMParser();
+    const doc: Document = parser.parseFromString(xmlString, "text/xml");
+    const result = doc.childNodes[0].textContent;
```

Comment:
nit - maybe fix potential bug here where doc doesn't have childNodes we'll crash with a non descriptive message.